worker_types/deepspeech-win/userdata

```diff
@@ -27,47 +27,35 @@ Set-ExecutionPolicy Unrestricted -Force -Scope Process
 # install chocolatey package manager
 Invoke-Expression ($client.DownloadString('https://chocolatey.org/install.ps1'))
 
-# install Windows SDK 8.1
-choco install -y windows-sdk-8.1
+# install Windows 10 SDK
+choco install -y windows-sdk-10.0
 
-# install June 2010 DirectX SDK for compatibility with Win XP
-$client.DownloadFile("http://download.microsoft.com/download/A/E/7/AE743F1F-632B-4809-87A9-AA1BB3458E31/DXSDK_Jun10.exe", "C:\DXSDK_Jun10.exe")
+# install NodeJS v8
+choco install -y nodejs --version 8.15.0
 
-# prerequisite for June 2010 DirectX SDK is to install ".NET Framework 3.5 (includes .NET 2.0 and 3.0)"
-Install-WindowsFeature NET-Framework-Core
-
-# now run DirectX SDK installer
-Start-Process C:\DXSDK_Jun10.exe -ArgumentList "/U" -wait -NoNewWindow -PassThru -RedirectStandardOutput C:\directx_sdk_install.log -RedirectStandardError C:\directx_sdk_install.err
-
-# install rustc dependencies (32 bit)
-$client.DownloadFile("http://download.microsoft.com/download/2/E/6/2E61CFA4-993B-4DD4-91DA-3737CD5CD6E3/vcredist_x86.exe", "C:\vcredist_x86-vs2013.exe")
-Start-Process "C:\vcredist_x86-vs2013.exe" -ArgumentList "/install /passive /norestart /log C:\vcredist_x86-vs2013-install.log" -Wait -PassThru
-
-# install rustc dependencies (64 bit)
-$client.DownloadFile("http://download.microsoft.com/download/2/E/6/2E61CFA4-993B-4DD4-91DA-3737CD5CD6E3/vcredist_x64.exe", "C:\vcredist_x64-vs2013.exe")
-Start-Process "C:\vcredist_x64-vs2013.exe" -ArgumentList "/install /passive /norestart /log C:\vcredist_x64-vs2013-install.log" -Wait -PassThru
+# install git
+choco install -y git --version 2.21.0
 
-# install more rustc dependencies (32 bit)
-$client.DownloadFile("http://download.microsoft.com/download/f/3/9/f39b30ec-f8ef-4ba3-8cb4-e301fcf0e0aa/vc_redist.x86.exe", "C:\vcredist_x86-vs2015.exe")
-Start-Process "C:\vcredist_x86-vs2015.exe" -ArgumentList "/install /passive /norestart /log C:\vcredist_x86-vs2015-install.log" -Wait -PassThru
+# install golang
+md "C:\gopath"
 
-# install more rustc dependencies (64 bit)
-$client.DownloadFile("http://download.microsoft.com/download/4/c/b/4cbd5757-0dd4-43a7-bac0-2a492cedbacb/vc_redist.x64.exe", "C:\vcredist_x64-vs2015.exe")
-Start-Process "C:\vcredist_x64-vs2015.exe" -ArgumentList "/install /passive /norestart /log C:\vcredist_x64-vs2015-install.log" -Wait -PassThru
+# set permanent env vars
+[Environment]::SetEnvironmentVariable("GOROOT", "C:\go", "Machine")
+[Environment]::SetEnvironmentVariable("PATH", $Env:Path + ";C:\gopath\bin", "Machine")
+[Environment]::SetEnvironmentVariable("GOPATH", "C:\gopath", "Machine")
```

Comment:
Note, you only need to install go if you are using it in your tasks. It isn't needed by generic-worker, so you can remove lines 39-45 if you want.

Reply:
But the `generic-worker` and `livelog` install after makes use of it, no?

Reply:
nevermind, just understood